src/components/subscription/subscription-section.tsx

```
@@ -0,0 +1,372 @@
+  };
+
+  const getConfirmationButtonText = (action: SubscriptionAction): string => {
+    switch (action) {
+      case SubscriptionAction.CANCEL:
+        return 'Cancel Subscription';
+      case SubscriptionAction.SUBSCRIBE:
+        return 'Subscribe';
+      case SubscriptionAction.REACTIVATE:
+        return 'Reactivate';
+      default:
+        const _exhaustiveCheck: never = action;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)